dask/array/core.py

```diff
@@ -769,7 +769,7 @@ def topk(self, k):
 
     def astype(self, dtype, **kwargs):
         """ Copy of the array, cast to a specified type """
-        return elemwise(partial(np.ndarray.astype, dtype=dtype, **kwargs),
+        return elemwise(partial(lambda x, dtype, **kwargs: x.astype(dtype, **kwargs), dtype=dtype, **kwargs),
```

Comment:
It seems odd to have a lambda within a partial.  Can these be fused to either a lambda or a partial